modules/auth/admin.go

```diff
@@ -28,6 +28,7 @@ func (f *AdminCreateUserForm) Validate(ctx *macaron.Context, errs binding.Errors
 // AdminEditUserForm form for admin to create user
 type AdminEditUserForm struct {
 	LoginType               string `binding:"Required"`
+	UserName                string
```

Comment:
Should be tagged with `binding:"Required;AlphaDashDot;MaxSize(35)"`. Otherwise there will be a 500 if the provided username is invalid